apps/mull-api/src/app/event/event.service.ts

```diff
@@ -12,34 +12,34 @@ export class EventService {
     private eventRepository: Repository<Event>
   ) {}
 
-  findAll(): Promise<Event[]> {
+  events(): Promise<Event[]> {
     return this.eventRepository.find();
   }
 
-  findOne(id: number): Promise<Event> {
+  event(id: number): Promise<Event> {
```

Comment:
Do we like this instead?
```suggestion
  findEvent(id: number): Promise<Event> {
```

Reply:
I'll be putting the name `getEvent` for the service method.